phpstan.neon.dist

```diff
@@ -0,0 +1,10 @@
+parameters:
+	level: 8
+	bootstrapFiles:
+		- .github/workflows/phpstan/autoload.php
+	paths:
+		- controllers
+		- ps_shoppingcart.php
+	ignoreErrors:
+		- '#^Call to an undefined method AdminController|FrontController\:\:#'
+		- '#^Call to an undefined method Module\:\:#'
```

Comment:
The file should be based on https://github.com/PrestaShop/php-dev-tools/blob/master/templates/phpstan/phpstan.neon

```
includes:
  - %currentWorkingDirectory%/vendor/prestashop/php-dev-tools/phpstan/extension.neon
```

is the most important part as it allows you to remove your own autoload.php

Reply:
The men, the hero :heart_eyes:

Reply:
Changed. I use the example in devdocs :heart_eyes: .